cmd/ocsp-responder/main.go

```diff
@@ -74,39 +111,15 @@ func (f *ocspFilter) check(req *ocsp.Request) error {
 	return nil
 }
 
-// newFilter creates a new ocspFilter which will filter out all requests for
-// certs which were not issued by one of the issuerCerts (here, paths to PEM
-// certs on disk) or which have a serial which does not start with one of the
-// given prefixes. The resulting filter will also reject all requests which
-// identify their issuer with a hash other than sha1.
-func newFilter(issuerCerts []string, serialPrefixes []string) (*ocspFilter, error) {
-	if len(issuerCerts) < 1 {
-		return nil, errors.New("Filter must include at least 1 issuer cert")
+// checkResponse returns a boolean if the the retrieved OCSP response is not
+// in fact for a certificate issued by the requested issuer (i.e. if the serial
+// is one we issued, but from a different issuer).
```

Comment:
```suggestion
// responseMatchesIssuer returns true if the CertificateStatus (from the DB) was generated
// by an issuer matching the issuerKeyHash from the OCSP request. This rejects,
// for instance, requests for a serial number that we issued, but from a different issuer
// than was indicated in the OCSP request.
```

(note the renaming of the method)